Update/omot_026.txt

```diff
@@ -686,7 +686,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 13, "ps3/s14/13/221300138", 256, TRUE);
 	OutputLine(NULL, "「月並みだが、迷ってる時間が惜しい。",
-			NULL, "\"I know this is cliche, but there's no time to waste making slow decicions.", Line_WaitForInput);
+			NULL, "\"I know this is cliché, but there's no time to waste making slow decisions.", Line_WaitForInput);
```

Comment:
cliché will be showed as cliche anyway, because Shift-JIS and font doesn't support é :P